src/js/views/patients/worklist/action_views.js

```diff
@@ -74,18 +74,24 @@ const ActionItemView = View.extend({
     Radio.trigger('event-router', 'patient:dashboard', this.model.get('_patient'));
   },
   onRender() {
-    this.showCheck();
-    this.showState();
-    this.showOwner();
-    this.showDueDate();
-    this.showDueTime();
     this.showForm();
     this.showDetailsTooltip();
+
+    this.canEdit = this.model.canEdit();
```

Comment:
technically we only need to update this if the owner changes..  we might decide to do that in the future as a perf improvement for long lists...